oca_port/utils/cache.py

```diff
@@ -171,6 +204,19 @@ def get_pr_from_commit(self, commit_sha: str):
             return self._commits_to_port["pull_requests"][str(pr_number)]
         return {}
 
+    def get_commit_files(self, commit_sha: str):
+        """Return file paths modified by a commit."""
+        return self._commits_data[commit_sha].get("files", set())
+
+    def set_commit_files(self, commit_sha: str, files: list):
+        """Store file paths modified by a commit."""
+        self._commits_data[commit_sha]["files"] = list(files)
+        try:
+            with self._commits_data_path.open(mode="w") as file_:
+                json.dump(self._commits_data, file_, indent=2)
+        except Exception:
+            pass
```

Comment:
I would like to store these data only once after the commit analysis, not on each analyzed commit to avoid IO.

Reply:
Done